### ISSUE TITLE
Each rank prints warning about nsplit

### ISSUE THREAD

**Issue by oksanaguba** [E3SM-Project/scream#1721]
I see a lot of warnings like below, i assume each rank prints them:
```
1: 172: Note: nsplit=-1, while nsplit must be >=1. We know SCREAM does not know nsplit until runtime, so this is fine.
1: 172:       Make sure nsplit is set to a valid value before calling prim_advance_subcycle!
```

**Comment by PeterCaldwell** [comment]
It would be nice if only masterproc printed this message. The file that writes this message is https://github.com/E3SM-Project/scream/blob/fcf28ecbac2c4d5aab7c727b4834f89433b20ae2/components/homme/src/theta-l_kokkos/cxx/cxx_f90_interface_theta.cpp#L69 . I'm not sure whether masterproc is easily accessible in this function. Any advice on what to do here?

**Comment by bartgol** [comment]
Unfortunately, that print statement is inside Homme. The only way to fix this is with a micro commit in homme (probably to be done in the upstream).

**Comment by ambrad** [comment]
I'll add a fix for this to my current PR to E3SM. I'll also likely bundle the fix for #1749 into that PR.

**Comment by ambrad** [comment]
FIxed in E3SM PR 5046
